api/mesh/v1alpha1/traffic_route.proto

```diff
@@ -228,6 +215,23 @@ message TrafficRoute {
     map<string, string> destination = 4;
   }
 
+  // List of selectors to match data plane proxies that are sources of traffic.
+  repeated Selector sources = 1
+      [ (validate.rules).repeated .min_items = 1, (doc.required) = true ];
+
+  // List of selectors to match services that are destinations of traffic.
+  //
+  // Notice the difference between sources and destinations.
+  // While the source of traffic is always a data plane proxy within a mesh,
+  // the destination is a service that could be either within or outside
+  // of a mesh.
+  repeated Selector destinations = 2
+      [ (validate.rules).repeated .min_items = 1, (doc.required) = true ];
```

Comment:
Seems like `doc.required` shouldn't be there anymore

Reply:
It also feels that the comment is not correct anymore. We can't really route to something outside the mesh no?